### ISSUE TITLE
How to run discriminator on generated images?

### ISSUE THREAD

**Issue by nvnnghia** [taki0112/StyleGAN-Tensorflow#15]
I am trying to run discriminator on generated images, But I couldn't make it working.
I tried something like:
`seed = np.random.randint(low=0, high=10000) 
test_z = tf.cast(np.random.RandomState(seed).normal(size=[self.batch_size, self.z_dim]), tf.float32)
 
alpha = tf.constant(0.0, dtype=tf.float32, shape=[])
 
self.fake_images = self.generator(test_z, alpha=alpha, target_img_size=self.img_size, is_training=False)

samples = self.sess.run(self.fake_images)

logit = self.discriminator(samples, alpha, self.img_size)

aa = self.sess.run(logit)

print(aa)
`
This code does not work. Please give me suggestions. Thank you
